client/src/components/carousel/carousel.jsx

```diff
@@ -41,12 +44,12 @@ class Carousel extends React.Component {
   }
   
   render () {
-    console.log('position: ' + this.state.position);
+    const carouselLength = this.props.images.length * 400 - 500;
     return (
       <div className={styles.container}>
         <div className={styles.scroll_container}>
-          <button className={`${styles.button} ${styles.left_button}`} onClick={() => this.clickLeft()}></button>
-          <button className={`${styles.button} ${styles.right_button}`} onClick={() => this.clickRight()}></button>
+          {(this.state.position === 0) ? null : <div className={`${styles.arrowbutton} ${styles.left_button}`} onClick={() => this.clickLeft()}><ChevronLeftIcon /></div>}
+          {(this.state.position >= carouselLength) ? null : <div className={`${styles.arrowbutton} ${styles.right_button}`} onClick={() => this.clickRight()}><ChevronRightIcon /></div>}
```

Comment:
you don't need to use the ternary if one is null, instead you can conditionally render with &&.  Ex: {(this.state.position !== 0) && <Component />}